src/particles/elements/diagnostics/openPMD.cpp

```diff
@@ -198,6 +199,7 @@ namespace detail
 #   endif
                 , "adios2.engine.usesteps = true"
             );
+            series.setSoftware("impactx", IMPACTX_VERSION);
```

Comment:
For excellent spelling :D
```suggestion
            series.setSoftware("ImpactX", IMPACTX_VERSION);
```